spec/bin/smartdown_spec.rb

```diff
@@ -0,0 +1,58 @@
+require 'spec_helper'
+require 'pathname'
+
+RSpec.describe 'bin/smartdown' do
+  let(:executable_path) {
+    Pathname.new('../../bin/smartdown').expand_path(File.dirname(__FILE__))
+  }
+
+  def fixture(name)
+    File.dirname(__FILE__) + "/../fixtures/acceptance/#{name}/#{name}.txt"
+  end
+
+  let(:input) { fixture("animal-example-simple") }
+
+  before(:each) do
+    @output = `#{executable_path} "#{input}" #{responses} 2>&1`
+    fail(raw_output) unless $?.success?
```

Comment:
Can we swap the cryptic `$?` for `$CHILD_STATUS`? You should just need to [`require 'english'`](http://ruby-doc.org/stdlib-2.1.4/libdoc/English/rdoc/English.html) beforehand.